site/scripts/db-indexes.ts

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable import/first */
+import dotenv from "dotenv";
+
+dotenv.config({ path: ".env.local" });
+import { User, UserDocument } from "../src/lib/model/user.model";
+import {
+  VerificationCode,
+  VerificationCodeDocument,
+} from "../src/lib/model/verificationCode.model";
+import { connectToDatabase } from "../src/lib/mongodb";
+import { EntityType } from "../src/lib/model/entityType.model";
+
+const catchAndLog = async (func: () => Promise<any>) => {
+  try {
+    await func();
+  } catch (error) {
+    if (error instanceof Error) {
+      console.error(`❌ Error trying to create index, ${error.toString()}`);
+    }
+  }
+};
+
+// Actions done in the following script should be _idempotent_ as they run in dev/prod
+void (async () => {
```

Comment:
Minor suggestion:

```ts
const script = async () => {
 // ...
}

void script();
```

This will be similar to what we have in `next-app/pages/api/*` with `handler` functions.

When we upgrade to native ESM modules, we’ll be able to replace `void script()` with `await script()` thanks to top-level await. Thus, we’ll be able to compose the scripts without using exports. That’s because top-level await [does not timeout](https://stackoverflow.com/q/68694714/1818285).

Until we‘ve unlocked top-level await, defining `const script` can help us composing them if needed. If we need to call a script from another script, we’ll just add `export default script` at the end of the file. 

I’m already using ESM scripts with top-level await [in a side project](https://github.com/kachkaev/tooling-for-how-old-is-this-house) and they are quite cool! Hope we’ll get there soon, but at least let’s get prepared 🙂